src/index.css

```diff
@@ -11,3 +8,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  width: 100%;
+  margin: 0;
```

Comment:
Here you reset the margin to 0, but you forget to add some margin or padding to the elements later on, for specifically mobile view. It gets a little crowded near the edge of smaller devices 🥰